yarn.lock

```diff
@@ -7739,7 +8202,7 @@
   dependencies:
     "@types/react" "*"
 
-"@types/react@*", "@types/react@>=16", "@types/react@^18.2.6":
+"@types/react@*", "@types/react@^18.2.6":
```

Comment:
There are now TypeScript errors, and they seem to be failing on class components (whether in our project or dependencies). It could possibly come from this version change for `@types/react`, but I lack evidence to back that hypothesis. When I ran `yarn why @type/react` in this branch and main, the top result stayed the same. The odd entry present in both of them is this:

```
=> Found "@types/react-dom#@types/react@17.0.59"
```

which is probably due to this strange mismatch in types that we've allowed to persist:

```json
"@types/react": "^18.2.6",
"@types/react-dom": "^17.0.10",
```

Reply:
Note that changing these entries to the following DOES NOT fix it:

```json
"@types/react": "^17.0.39",
"@types/react-dom": "^17.0.11",
```